Package

```diff
@@ -8,25 +8,24 @@ plugin_install_dirname=${plugin_name}
 #   1. Generic packages we depend on in all architectures and package managers
 #   2. Architecture specific packages we depend on
 #   3. Package manager specific packages we depend on
-requirements="foglamp,wiringpi"
+requirements="foglamp"
 
 case "$arch" in
-	x86_64)
-		;;
-	armhf)
-		;;
-	aarch64)
-		;;
+    x86_64)
+        echo "${plugin_name} is not supported on Intel architectures as it requires ARM GPIO support"
+        exit 1
+        ;;
+    armv7l)
+        requirements="${requirements},wiringpi"
+        ;;
+    aarch64)
```

Comment:
@MarkRiddoch Kindly advise on requirements for `aarch64` seems like _wiringpi_ package is not available.

Reply:
Yes, the same as a RPi

Reply:
`wiringpi` package is not available on `aarch64`. We need to find out the way to install on coral board.

Reply:
OK, this is not a priority as it is doubtful anybody is gong to be using a Coral board for this. Let's keep it as a background thing.